spec/services/reject_application_by_default_spec.rb

```diff
@@ -29,6 +29,22 @@ def create_application
     expect(application_choice.reload.rejected_by_default).to be true
   end
 
+  it 'sets `rejected_at`' do
+    application_choice = create_application
+    described_class.new(application_choice: application_choice).call
+    expect(application_choice.reload.rejected_at).not_to be_nil
+  end
+
+  it 'calls SetDeclineByDefault service' do
```

Comment:
Probably best to do the same as you did in `make_an_offer_spec.rb` here.

Reply:
ok

Reply:
Ha, can't do that, because calling SetDeclineByDefault on a rejected application has no side-effects for that application (but may trigger DBD to be set on another, if no choices remain awaiting_provider_decision).